package/MDAnalysis/coordinates/base.py

```diff
@@ -1589,6 +1608,24 @@ class Writer(IObase):
     See Trajectory API definition in :mod:`MDAnalysis.coordinates.__init__` for
     the required attributes and methods.
     """
+    class __metaclass__(type):
+        # Auto register upon class creation
+        def __init__(cls, name, bases, classdict):
+            type.__init__(type, name, bases, classdict)
```

Comment:
It should be harmless in this case. I wasn't sure what the answer to this is, but I'm fine with leaving it.